### ISSUE TITLE
Package.getStylesheetsPath is deprecated.

### ISSUE THREAD

**Issue by uvlek** [mark-hahn/web-browser#47]
Store package style sheets in the `styles/` directory instead of `stylesheets/` in the `web-browser` package
```
Package.getStylesheetsPath (C:\Users\vetka\AppData\Local\atom\app-0.189.0\resources\app\src\package.js:448:9)
Package.getStylesheetPaths (C:\Users\vetka\AppData\Local\atom\app-0.189.0\resources\app\src\package.js:459:32)
```

**Comment by Aarilight** [comment]
@mark-hahn this deprecation can be fixed by simply removing the /stylesheets directory... did you leave it for backwards compatibility?

**Comment by mark-hahn** [comment]
No.  I just haven't gotten around to it.  I'm hoping to go through all my
packages in the next few days and remove deprecations.  It's not going to
be easy with 20 packages. (grin)

On Wed, Apr 8, 2015 at 10:59 PM, Mackenzie McClane <notifications@github.com

**Comment by mark-hahn** [comment]
Fixed by PR #48
